src/LcdMenu.cpp

```diff
@@ -343,9 +312,9 @@ void LcdMenu::printMenu(String line)
 
     _lcd.setCursor(_activeCol, _charHeightRows*_activeRow);
     int spaces = _columns - line.length();
-    for (unsigned int i = 0; i < line.length(); i++)
+    for (char i : line)
```

Comment:
What is this voodoo magic syntax? I must have missed a C++ standard along the way, lol. Nice.

Reply:
Yep! Range based for loops was added in C++11 ([docs](https://en.cppreference.com/w/cpp/language/range-for)) and it can really clean up simple string loops like this.